web/templates/package/index.html.eex

```diff
@@ -71,17 +71,37 @@
 
 <div class="clearfix"></div>
 
+
+<%=  if @exact_match do %>
+  <div class="exact-match">
+    <h5>Exact Match:</h5>
+    <ul>
+      <li>
```

Comment:
could you extract `<li>...</li>` into a `_package.html`? we could reuse it for displaying exact match and a normal package as it's exactly the same